scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/LiveJobs.java

```diff
@@ -106,9 +111,18 @@ void unlock() {
 
     private final StartAtUpdater startAtUpdater = new StartAtUpdater();
 
-    LiveJobs(SchedulerDBManager dbManager, SchedulerStateUpdate listener) {
+    private final SynchronizationInternal synchronizationInternal;
+
+    private static final String SIGNAL_ORIGINATOR = "SCHEDULER";
```

Comment:
better use "scheduler" in lowercase, as it corresponds to the actual scheduler user.